ravenpy/models/aggregate_forcings_to_HRUs.py

```diff
@@ -0,0 +1,232 @@
+from ravenpy.models.importers import (
+    RoutingProductGridWeightImporter,
+    GridWeightsCommand
+)
+import netCDF4 as nc4
+import numpy as np
+
+
+DIM_NAMES = ("lon", "lat")
+VAR_NAMES = ("lon", "lat")
+ROUTING_ID_FIELD = "HRU_ID"
+NETCDF_INPUT_FIELD = "NetCDF_col"
+AREA_ERROR_THRESHOLD = 0.05
+VARIABLES_TO_AGGREGATE = ["Streaminputs"]
```

Comment:
That's an unexpected default value from my POV. I suspect that in the majority of cases, users will aggregate forcing fields (pr, tas). I suggest that the default variable are left as dynamic and determined by the variables that are defined over dimensions that are a subset of DIM_NAMES. 
See https://github.com/pangeo-data/xESMF/pull/79/files for an example of this logic in action.